hack/install-tools.sh

```diff
@@ -10,5 +10,6 @@ echo "Detected go version: $(go version)"
 go get github.com/tools/godep
 go get github.com/onsi/ginkgo/ginkgo  # installs the ginkgo CLI
 go get github.com/onsi/gomega         # fetches the matcher library
+go get github.com/ilackarms/webdavclnt # required package for testing
```

Comment:
@ilackarms Are you sure you meant to use this repo?

Reply:
this is necessary because the upstream version doesn't support setting headers on the request (we need the auth token to be set). see https://github.com/openshift/image-inspector/pull/77/files#diff-bdd65710e10fae871f7486e54fd9a919R212